core/driver/sources/src/main/scala/org/virtuslab/ideprobe/config/PathsConfig.scala

```diff
@@ -1,12 +1,12 @@
 package org.virtuslab.ideprobe.config
 
-import java.nio.file.Path
+import java.nio.file.{Path, Paths}
 
 case class PathsConfig(
     base: Option[Path] = None,
     instances: Option[Path] = None,
     workspaces: Option[Path] = None,
-    screenshots: Option[Path] = None,
+    screenshots: Option[Path] = sys.env.get("IDEPROBE_SCREENSHOTS_DIR").map(path => Paths.get(path)),
```

Comment:
Hmmm okay... now we still have an extraction of `IDEPROBE_SCREENSHOTS_DIR` in `org.virtuslab.ideprobe.Screenshot#outputDirectory` as well... does it even have any effect now?

Reply:
Also, intuitively this doesn't feel like a good place to retrieve this env var... wouldn't `org.virtuslab.ideprobe.IdeProbePaths#from` be more consistent/expected?

Reply:
yes without getting here this env,  `IdeProbePaths` set default screenshot directory as env and in effect this default env is extracted in `org.virtuslab.ideprobe.Screenshot` (this is how the code worked before)

Reply:
Damn. Screenshot mechanism was evolving an there is a bit of mess that I see now. Our CI tries to override screenshots dir using env var that is picked up by IntelliJ and set by driver. Additionally reading it in this place would likely fix it, but i am not sure if this is the clean solution.

You may explicitly configure probe to pickup this from env var, by writing this:
```
probe.paths.screenshots = ${?MY_CI_SCREENSHOTS_DIR}
```
If such env var will be set, probe will use it, if not, it will use the default (thanks to the `?`).

For non .conf scenario, user would call `fixture.withPaths(IntelliJPaths(/*construct the instance passing in env var for screenshots path*/))`.

Having a separate screenshot dir magic env var that probe will pick up automatically is a separate thing to consider, maybe it makes sense - at least is very easy to use.

Reply:
Oh! okay

Know what... TBH I'd completely remove access to any env vars from `Screenshot`. This is mixing of responsibilities (retrieving config **and** doing a screenshot) and confusing as hell (esp. if non-trivial threading is taken into account). 

Let's pass all the needed config to `Screenshot` c'tor from outside for better clarity

Reply:
(removed the comment, haven't seen Łukasz's one)

Reply:
If possible, I'd completely remove access to any env vars from `Screenshot`. This is mixing of responsibilities (retrieving config and doing a screenshot) and confusing as hell (esp. if non-trivial threading is taken into account).

Let's pass all the needed config to Screenshot c'tor from outside for better clarity

Reply:
Oh, I just noticed it is not as messy as I thought, it is even configured as I explained in this file `examples/src/test/resources/ideprobe.conf`. However work on these `examples` was never completed. So the only tests that could run with potentially working screenshots mechanism, are disabled xd

Reply:
> Our CI tries to override screenshots dir using env var that is picked up by IntelliJ and set by driver

Yeah, this is pretty... surprising 😅

```
// src/test/resources/ideprobe.conf
probe {
  // ...
  paths.screenshots = ${?IDEPROBE_SCREENSHOTS_DIR}
}
```

> Additionally reading it in this place would likely fix it, but i am not sure if this is the clean solution.

As suggested above... maybe `org.virtuslab.ideprobe.IdeProbePaths#from` for reading all such config?

> Having a separate screenshot dir magic env var that probe will pick up automatically is a separate thing to consider, maybe it makes sense

Huh... so `IDEPROBE_SCREENSHOTS_DIR` was NOT meant to be such a env var, and that's why it wasn't documented in the first place? 😅 if so, then maybe there wasn't any bug here in the first place 🐞

Reply:
I think this mess was introduced during refactoring that introduced the config for screenshot path. It was supposed to replace the less flexible env var. 
Regardless of that, screenshots do not work in this repo.

Reply:
This is a thing we missed. Originally we had implemented ide-probe to work with jUnit. Screenshot mechanism depends on that. It is completely possible that ide-probe is run outside of tests or using a different framework. In that case we will not detect and setup test suite and test case env vars in InstalledIntelliJ, but  we should still take screenshots. Solution would be to implement Screenshot#outputDirectory differently.

Reply:
do you think about the option of e.g. deletion `TEST_SUITE` and `TEST_CASE `from `Screenshot#outputDirectory` and save everything directly to one folder `IDEPROBE_SCREENSHOTS_DIR` ? (or additionally some optional parameter if user wants to save some screenshot in special subfolder under  `IDEPROBE_SCREENSHOTS_DIR`)

Reply:
More like: if the test data is not available save them right in the screenshots dir and if it is available, save them as before (dirs according to suite and name).
This is a minimal solution to get screenshots working. Having optional subdirectory path or more control over name is a separate thing, and if there is a use case for that it is worth thinking how to do it best.

Reply:
do you anticipate the possibility of setting TEST_SUITE and TEST_CASE  by the ide-probe user ? or is this behaviour not recommended (mean not to solve case when user set their own TEST_SUITE and TEST_CASE)?